tests/unit/fetchFreePlanVerifiedBankAccountTest.js

```diff
@@ -139,19 +121,7 @@ describe('getCurrentStep', () => {
                     },
                 }],
                 requestorAddressCity: 'Portland',
-                verifications: {
-                    externalApiResponses: {
-                        realSearchResult: {
-                            status: 'pass',
-                        },
-                        lexisNexisInstantIDResult: {
-                            status: 'pass',
-                        },
-                        requestorIdentityID: {
-                            status: 'pass',
-                        },
-                    },
-                },
+                verifications: {},
```

Comment:
NAB, can't we just remove this altogether if we're not expecting anything in it?